js/Cardz.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import {connect} from 'react-redux';
+
+import * as actions from '../redux/actions/workspace.js';
+import store from '../redux/store.js';
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card';
+import update from 'react-addons-update';
+
+const styles = {
+    cardz: {
+        maxHeight: 500
+    }
+};
+
+
+class Cardz extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {tilesData: []};
+};
+
+  handleToggle(toggle, index, event) {
+    var newState = update(this.state, {tilesData: {[index]: {expanded: {$set: toggle}}}})
+    console.log(index)
+
+    this.setState(newState);
+  };
+
+  componentWillMount() {
+      this.props.dispatch(actions.fetchWorkspaceData()); // puts the data in the store
+
+
+
+  };
+
+updateCard() {
+    var newState = update(this.state, {
+        tilesData: {
+            [index]: {
+                placeId: {
+                    $set: this.props.actions.workspace.currentPlace
+                }
+            }
+        }
+    })
+    this.setState(newState);
+};
+
+  render() {
+    var tiles = this.props.state.workspaceCache.map((tile, index) => {
+      console.log('strung: ', tile);
```

Comment:
Try to clean up old debug printing from commits wherever possible.  It obscures more important things in the logs.